bootloader/bootloader.go

```diff
@@ -144,6 +144,9 @@ type ExtractedRunKernelImageBootloader interface {
 type ManagedAssetsBootloader interface {
 	// IsCurrentlyManaged returns true when the on disk boot assets are managed.
 	IsCurrentlyManaged() (bool, error)
+	// BootAssets returns a list of boot assets managed by the bootloader in
+	// the boot filesystem.
+	BootAssets() []string
```

Comment:
this should be called ManagedAssets I think, to match the other method and to make clear that it lists only the managed ones, not all the assets related to booting with this bootloader